Sources/SwiftDocC/Model/Rendering/Symbol/AvailabilityRenderMetadataItem.swift

```diff
@@ -163,3 +163,27 @@ public struct AvailabilityRenderItem: Codable, Hashable, Equatable {
         self.isBeta = isBeta
     }
 }
+
+extension Array where Array.Element == AvailabilityRenderItem {
```

Comment:
FIY: this can be expressed more simply as
```suggestion
extension Array<AvailabilityRenderItem> {
```